tools/fapi/tss2_authorizepolicy.c

```diff
@@ -73,10 +73,11 @@ int tss2_tool_onrun (FAPI_CONTEXT *fctx) {
         policyRefSize);
     if (r != TSS2_RC_SUCCESS){
         LOG_PERR ("Fapi_AuthorizePolicy", r);
+        if (policyRef) free (policyRef);
```

Comment:
free (policyRef);

free (NULL) is a no-op, so you don't need to check.

Reply:
Fixed in updated PR.